### ISSUE TITLE
Unicode characters in password allow for less than seven character password

### ISSUE THREAD

**Issue by leonard-IMBERT** [xvw/muhokama#33]
Not sure if this _really_ is an issue but by using unicode characters (like 🬃) you can bypass the seven characters requirements in the account creation.

# Why ?

It seems that a certain range of unicode characters are encoded in 8 bytes instead of the orignal 4. The `password` type of the `<input>` understand characters as their UTF-16 representation (source: [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/password#minlength)) meaning that for the input "🬃" is two character long.

I don't think there is an easy fix, it comes from the browser interpretation of unicodes characters. This is more of a trivia than really an issue but I thought it might be useful to keep this information around

**Comment by gr-im** [comment]
Yes, I have a precise idea about how to fix that (and it will serve the category failure too!) thanks for your consciencious beta-testing!
